args.go

```diff
@@ -134,4 +134,6 @@ const (
 
 	// ArgResourceType is the resource type for snapshot.
 	ArgResourceType = "resource"
+	// ArgSnapshotDesc is the description for volume snapshot.
+	ArgSnapshotDesc = "snapshot-desc"
```

Comment:
Sorry tiny nitpick I just noticed, could we move this up to under `ArgSnapshotName` to make it a little easier to find? https://github.com/digitalocean/doctl/pull/173/files#diff-07ee57a49d1c0b692bd79676cc31dd6fR54

I'll merge this PR after that.